ocs_ci/utility/ocs_build.py

```diff
@@ -0,0 +1,66 @@
+"""
+This module is used to return latest OCS internal build for specified OCS
+version.
+"""
+
+import argparse
+import os
+
+from ocs_ci.framework import config
+from ocs_ci.framework.main import load_config
+from ocs_ci.ocs.constants import OCS_VERSION_CONF_DIR
+from ocs_ci.utility.utils import get_latest_ds_olm_tag
+
+
+def init_parser():
+    """
+    Init argument parser.
+
+    Return:
+        object: Parsed arguments
+    """
+
+    parser = argparse.ArgumentParser(
+        description="OCS Internal build version",
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+    parser.add_argument(
+        "--ocs-version",
+        action="store",
+        required=False,
+        default=config.ENV_DATA["ocs_version"],
+        help=f"""
+            OCS version in format X.Y (e.g. 4.7). If not specified, the default
+            value {config.ENV_DATA['ocs_version']} will be used.
+        """,
```

Comment:
As mentioned on chat, this slightly breaks formatting of the help function (it add additional indentation for this one parameter) and looks like this:
```
  -h, --help            show this help message and exit
  --ocs-version OCS_VERSION
                        
                                    OCS version in format X.Y (e.g. 4.7). If not specified, the default
                                    value 4.7 will be used.
                                
  --image               If used the whole image of OCS internal build will be returned
```

Reply:
Removed this option